### ISSUE TITLE
Version 2.3.1 won't start on macOS 14.5

### ISSUE THREAD

**Issue by breun** [christofmuc/KnobKraft-orm#329]
Version 2.3.0 works for me on macOS 14.5 (23F79), but I can't get 2.3.1 to launch. I ran `xattr -cr /Applications/KnobKarftOrm.app`, because without that macOS won't even attempt to launch the app (I guess because it's not signed), but after that, the app still won't launch.

It looks like 2.3.1 just immediately exists, I'm not getting any error messages.

Try to launch it from the command line also doesn't show any more information:

```
❯ /Applications/KnobKraftOrm.app/Contents/MacOS/KnobKraftOrm 
zsh: killed     /Applications/KnobKraftOrm.app/Contents/MacOS/KnobKraftOrm
```

**Comment by christofmuc** [comment]
Thanks for pointing that out! I am surprised but the update of the macOS version also changed CPU architecture of the github build machine. So the 2.3.1 was actually built for ARM architecture, and those no longer accept unsigned executables. They do accept unsigned x86 images running them with Rosetta for now, so I guess at some point I will have to bite the bullet and get the signing/notarization process going.

For now, I have reverted the build machine back to macOS 13, which should give me an x86 build, which should run as usual after RIGHT-Click/CTRL-Click and Open override, or using xattr as you mentioned.

I got a crash report running the build on my M1 which said "invalid signature".

pipeline for 2.3.2 is running right now, will update once its done.
